docs/website/docs/examples/_examples-header.md

```diff
@@ -0,0 +1,22 @@
+import Admonition from "@theme/Admonition";
+import CodeBlock from '@theme/CodeBlock';
+
+<Admonition>
+    The <span>{props.title}</span> Example is part of the <a href="/docs/examples">DLT Code Examples</a>, a list of comprehensive examples to help you solve real world loading problems with dlt.
```

Comment:
dlt not DLT. please!

Reply:
alright, I have fixed this in a few other places I have found.